### ISSUE TITLE
React Native 0.63 upgrade (Xcode)

### ISSUE THREAD

**Issue by curiousdustin** [react-native-community/upgrade-support#110]
The release notes send you to the [upgrade helper](https://react-native-community.github.io/upgrade-helper/), which is great, except, no human could possibly follow the diff log for the xcode project. 

Why aren't the steps to upgrade the xcode project documented?!

How is anyone supposed to be confident in their upgrade?

#13 is great! Thanks @pvinis. 

Can anyone share similar documentation for 0.63.x?

**Comment by JB-CHAUVIN** [comment]
Yes, can anyone specify the upgrade steps needed with XCODE ? On react native 0.63.3.
Thanks.

**Comment by williamgurzoni** [comment]
Yes man, it'll be great to have this tutorial 👍

**Comment by curiousdustin** [comment]
Same issue with the 0.64.0 release.

How are we supposed to figure out how to edit our Xcode projects?!

Apparently, this sort of documentation is on the [list of things to do](https://github.com/react-native-community/releases/issues/214) for a given release, but... just never done?

>  Create Xcode project upgrade guide to link to from the Upgrade Helper

**Comment by gomdolkim** [comment]
I've tried to find out which xcode version I have to use for 0.63.3. and can't find any documentation. anyone got it before?
